.travis.yml

```diff
@@ -2,7 +2,7 @@ language: node_js
 node_js:
   - stable
 script:
-  - npm run lint && npm run csslint && npm run test && npm run downloadsolc && npm run make-mock-compiler && npm run build
+  - npm run lint && npm run test && npm run downloadsolc && npm run make-mock-compiler && npm run build
```

Comment:
We've only ignored the less severe warnings. And that the ignore list should be shrunk too as we fix those less severe warnings, just nobody bothered.

It did help us catch a ton of severely invalid CSS when I've added it.